.php_cs.dist

```diff
@@ -62,10 +65,16 @@ return PhpCsFixer\Config::create()
         ],
         'no_useless_else' => true,
         'no_useless_return' => true,
-        'ordered_imports' => true,
+        'ordered_imports' => [
+            'importsOrder' => [
+                'class',
+                'function',
+                'const',
+            ],
+            'sortAlgorithm' => 'alpha',
+        ],
         'phpdoc_order' => true,
-        'psr4' => true,
-        'semicolon_after_instruction' => true,
+        // 'simplified_null_return' => true,
```

Comment:
We may be able to use this rule once we require at least PHP 7.1 and have the nullable return type declarations in place. But it might not be possible to have a return type declaration in all cases...

Reply:
note that master requires >=7.1

Reply:
I'm aware of that. Will try turning this rule on in `master`.